### DISCUSSION TITLE
Suggested/Recommended Desktop Manager

### DISCUSSION THREAD

**Discussion by edwardp3** [lxqt/lxqt#2097] [Q&A]
I was unsuccessful trying to configure SDDM, which is the default DM for Debian's LXQt image, only to discover that there was a bug report filed with Debian on the same issue _over three years ago_, which to this day has not been resolved. 

Is there a suggested or recommended desktop manager for LXQt that would be better than SDDM and is easy to configure, preferably via a GUI? I know the LXDE side has LXDM, but I've read that it is no longer maintained.

Thanks in advance.

**Comment by stefonarch** [comment]
What did you try to configure? Afaik only issue is the autologin in sddm-config-editor, which can be solved by any text editor.
There is another GUI to configure it but as downside it installs plasma-desktop.
`kcmshell5 --caption "LXQt KWin Settings" --desktopfile=/usr/share/kservices5/kwindecoration.desktop kcm_sddm

Sddm is still the best choice IMO.
`

**Reply by tsujan** [reply]
> Sddm is still the best choice IMO.

I agree.

That being said, any desktop manager can be used with LXQt. Different users may prefer different DMs.

**Reply by stefonarch** [reply]
Because I didn't realize that autologin was broken by sddm-config-editor I siwtched to no display manager, using tty, startx and autologin:
https://wiki.archlinux.org/title/Getty#Automatic_login_to_virtual_console
